tardis/montecarlo/tests/test_packet_source.py

```diff
@@ -75,3 +93,47 @@ def test_bb_packet_sampling(
         assert np.all(np.isclose(nus, ref_df["nus"]))
         assert np.all(np.isclose(mus, ref_df["mus"]))
         assert np.all(np.isclose(unif_energies, ref_df["energies"]))
+
+    def test_bb_packet_sampling_relativistic(
+        self,
+        tardis_ref_data,
+        blackbody_simplesource_relativistic,
+    ):
+        """
+        Parameters
+        ----------
+        tardis_ref_data : pd.HDFStore
+        blackbody_simplesource_relativistic : tardis.montecarlo.packet_source.BlackBodySimpleSourceRelativistic
+        """
+        blackbody_simplesource_relativistic.temperature = 10000
+        blackbody_simplesource_relativistic.beta = 0.25
+
+        nus = blackbody_simplesource_relativistic.create_packet_nus(100)
+        unif_energies = (
+            blackbody_simplesource_relativistic.create_packet_energies(100)
+        )
+        blackbody_simplesource_relativistic._reseed(2508)
+        mus = blackbody_simplesource_relativistic.create_packet_mus(10)
+
+        gamma = np.sqrt(1 - 0.25**2) ** -1
+        ref_df = tardis_ref_data["/packet_unittest/blackbody"]
+        expected_nus = ref_df["nus"]
+        expected_unif_energies = ref_df["energies"] * 1.6 / gamma
```

Comment:
Can I get a reminder of what the 1.6 represents? I did look through all this a few months ago, but I have since forgotten many of the details, so a quick reminder or explanation would be nice!

Reply:
1.6 is the "static_inner_boundary2cmf_factor" , which is calculated as (2 * beta + 1) / (1 - beta**2). The factor appears because the inner boundary is not comoving with the material (contrary to what one would intuitively assume). If you think it is better, I can calculate the factor here instead of hardcoding it. The idea behind hardcoding it was to avoid code duplication between the test and the method being tested.

Reply:
I think if it is explicit somewhere else then it's probably fine. You're the expert -- if it makes sense to do it this way, then that's good enough for me!